Server/sprawl_calculator/calculator/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+#Url for endpoint1 and endpoint2
+urlpatterns = [
+    path('v1/wup_index>',endpoint1, name='endpoint1'),
+    path('v1/wup_index/<processing_index>',endpoint2, name='endpoint2'),
```

Comment:
Lets rename the second paramter currently `endpoint2` -> `wup_index_generation_status`

Reply:
As we renamed index_calculation_result to  index_calculation_status, same is updated in urls also